examples/ui/ui_material.rs

```diff
@@ -0,0 +1,70 @@
+//! This example illustrates how to create a Node that shows a Material and driver parameters from
```

Comment:
```suggestion
//! This example illustrates how to create a Node that shows a Material and derives parameters from
```

"driver" meant to be "derives"?

Reply:
I wanted to write "drives" as in "one can use a system that drives the parameters of this material".

I don't think its a great description, but can't seem to find the words for a better one right now.